src/commands/xcode-open

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+
+error(){
+  red="\e[1;31m"
+  reset="\e[0m"
+  printf "${red}%s${reset}\n" "$1"  >&2
+}
+
+info(){
+  printf "%s\n" "$1"
+}
+
+usage='xcode-open
+Open a directory with Xcode
+
+usage: xcode-open [-h | --help] [--current | --beta] [PATH]
+
+  -h | --help           Print this usage documentation and exit
+
+  --current             DEFAULT - Use the currently selected Xcode version.
+
+  --beta                Use the newest beta version of Xcode.
+
+  PATH                  The path to the directory containing the Xcode project.
+                        If not provided, will default to the current directory.
+'
+
+app=$(xcode-version --path)
+
+case $1 in
```

Comment:
Exactly. I also want to provide long switches for the sake of readability/scripting.